src/Build/Collections/RetrievableEntryHashSet/HashSet.cs

```diff
@@ -92,8 +92,6 @@ internal class RetrievableEntryHashSet<T> : ICollection<T>,
     {
         // store lower 31 bits of hash code
         private const int Lower31BitMask = 0x7FFFFFFF;
-        // cutoff point, above which we won't do stackallocs. This corresponds to 100 integers.
```

Comment:
Revert entire file please. It's designed as minimal-edits from a framework type, see

https://github.com/Microsoft/msbuild/blob/0d29906ea866b5e6d79d77478d36b14c13e0a4d3/src/Build/Collections/RetrievableEntryHashSet/HashSet.cs#L36-L39